lighthouse-cli/test/smokehouse/perf/expectations.js

```diff
@@ -79,8 +79,8 @@ module.exports = [
               {resourceType: 'font', requestCount: 2, size: '80000±1000'},
               {resourceType: 'script', requestCount: 3, size: '55000±1000'},
               {resourceType: 'image', requestCount: 2, size: '28000±1000'},
-              {resourceType: 'document', requestCount: 1, size: '2100±100'},
-              {resourceType: 'other', requestCount: 2, size: '1250±50'},
+              {resourceType: 'document', requestCount: 1, size: '2200±100'},
```

Comment:
These numbers went up because `Content-Type` either was added where it previously wasn't present, or it got larger.

All these files were just sending ascii, which doesn't encode larger in utf-8 than plain binary.